terraform/ci/main.tf

```diff
@@ -30,13 +30,14 @@ resource "random_id" "default" {
 
 
 locals {
-  static_bucket_name = "pmap-static-ci-bucket-${random_id.default.hex}"
+  ci_static_bucket_name        = "pmap-static-ci-bucket-${random_id.default.hex}"
```

Comment:
No need to add `ci` as prefix as this is terraform/ci/main.tf

Reply:
Removed.